src/audio.cpp

```diff
@@ -25,15 +25,15 @@ ILLIXR_AUDIO::ABAudio::ABAudio(std::string outputFilePath, ProcessType procTypeI
 
 ILLIXR_AUDIO::ABAudio::~ABAudio(){
     if (processType == ILLIXR_AUDIO::ABAudio::ProcessType::FULL){
-        free(outputFile);
+        delete outputFile;
     }
     for (unsigned int soundIdx = 0; soundIdx < soundSrcs->size(); ++soundIdx){
-        free((*soundSrcs)[soundIdx]);
+        delete soundSrcs->operator[](soundIdx);

```

Comment:
```suggestion
        delete (*soundSrcs)[soundIdx];
```
Imo, avoid using `operator` when possible.